docker/datahub-gms/Dockerfile

```diff
@@ -39,6 +39,13 @@ COPY --from=prod-build /datahub-src/docker/datahub-gms/start.sh /datahub/datahub
 COPY --from=prod-build /datahub-src/docker/monitoring/client-prometheus-config.yaml /datahub/datahub-gms/scripts/prometheus-config.yaml
 RUN chmod +x /datahub/datahub-gms/scripts/start.sh
 
+# Snappy Lib update
+RUN apk update && apk add --no-cache gcompat
```

Comment:
Minor comment. Can we just add this to line 29?

Reply:
@dexter-mh-lee As the snappy library should be available in the runtime image. Adding that to the build image (line 29) would not work.

Reply:
Got it!